### DISCUSSION TITLE
How to set the font color?

### DISCUSSION THREAD

**Discussion by GollyJer** [anday013/react-native-otp-entry#62] [Q&A] [answered]
Hi! This is an awesome component. Thanks of making it!

I can't get the font color to change. Wondering if it's just me?

I tried passing 
```javascript
textInputProps={{
  style: { color: 'red' },
}}
```
and also passing a stylesheet 
```javascript
const styles = StyleSheet.create({
  input: {
    color: 'red',
  },
});

textInputProps={{
  style: style.input,
}}
```
neither approach worked.

Thanks!

**Comment by anday013** [comment, marked as answer]
Hi, thank you!

For any UI modification, you can use the `theme` prop. An example of the font color change is here:

<img width="300" alt="image" src="https://github.com/anday013/react-native-otp-entry/assets/48630069/630d0fd8-43c8-4509-b7c9-c22bc128712b">

**Reply by GollyJer** [reply]
Ha. I was staring right at it. 😀

I looked at the source and saw `textInputProps?.style` being passed to the TextInput and got confused.
